src/adagrams.js

```diff
@@ -1,15 +1,107 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+  const LETTER_POOL = {
+    A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2,
+    I: 9, J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2,
+    Q: 1, R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1,
+    Y: 2, Z: 1
+  };
+  
+  const MAX_HAND_COUNT = 10;
+  let availableLetters = [];
+
+  for (let letter in LETTER_POOL) {
+    let originalLetterCount = LETTER_POOL[letter];
```

Comment:
We could use [`Object.entries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) as well to get the functionality and cut out the key accessing you have here like so: 
```javascript
for (const [letter, freq] of Object.entries(LETTER_POOL)){
    // ...
```